### PR TITLE
Update examples in the readme

### PR DESCRIPTION
**Pull request by mindreframer**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,39 +7,50 @@ var TalonOne = require('talon-one')
...
-client.updateCustomerSession(customerId, {
+
+client.updateCustomerSession(sessionId, {
   // associate this session with the profile we created above
-  profileId: profileId,
-  // set the currency being used in this session
-  currency: 'USD',
+  profileId: customerId,
+  // set referral ID for this session
+  referral: 'somereferral-identifier',
```
Comment (janv) [inline README.md]: Would this be something internal to the shopsystem or a referral code from our database?

File: README.md [hunk 2]
```diff
@@ -7,39 +7,50 @@ var TalonOne = require('talon-one')
...
-var sessionId = 'some-identifier-for-this-session'
 
-client.updateCustomerSession(customerId, {
+
+client.updateCustomerSession(sessionId, {
   // associate this session with the profile we created above
-  profileId: profileId,
-  // set the currency being used in this session
-  currency: 'USD',
+  profileId: customerId,
```
Comment (janv) [inline README.md]: is profileId synonymous with integrationId here?

Comment (mindreframer) [reply README.md]: yes, this is the integrationID for the customer_profile. I did not want to change the current API, so i have kept `profileId`

File: README.md (hunk 1 above)
Comment (mindreframer) [reply README.md]: this value is stored on the `referral` field in the customer_sessions table. I am not sure, if that is a referral code from our DB or something else...

**Pull request merged**